misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -164,16 +164,15 @@ public static boolean setModuleFromSyntaxTree(Module module, SyntaxTree syntaxTr
                         ((EnumDeclarationNode) node).qualifier().isPresent() &&
                         ((EnumDeclarationNode) node).qualifier().get().kind().equals(SyntaxKind.PUBLIC_KEYWORD)) {
                     module.enums.add(getEnumModel((EnumDeclarationNode) node));
-                } else if (node.kind() == SyntaxKind.MODULE_VAR_DECL &&
-                        ((ModuleVariableDeclarationNode) node).visibilityQualifier().isPresent() &&
-                        ((ModuleVariableDeclarationNode) node).visibilityQualifier().get().kind()
-                                .equals(SyntaxKind.PUBLIC_KEYWORD)) {
+                } else if (node.kind() == SyntaxKind.MODULE_VAR_DECL) {
                     DefaultableVariable defaultableVariable = getModuleVariable((ModuleVariableDeclarationNode) node,
                             semanticModel, module);
                     if (containsToken(((ModuleVariableDeclarationNode) node).qualifiers(),
                             SyntaxKind.CONFIGURABLE_KEYWORD)) {
                         module.configurables.add(defaultableVariable);
-                    } else {
+                    } else if (((ModuleVariableDeclarationNode) node).visibilityQualifier().isPresent() &&
```

Comment:
Since we are modifying the code here anyway, shall we also extract this cast to a separate variable? I can see the same cast repeated multiple times in the vicinity.

Reply:
@lochana-chathura addressed review suggestions